compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/Configurations.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.compiler.internal.parser;
+
+import java.util.Base64;
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+public final class Configurations {
```

Comment:
Shall we rename to`ParserConfigurations`

Reply:
Updated 013d41a7fd8ac5f77bfdee99b2e19b03b3e2312a